Source/FacilityManagementWebsite/SmartHotel.FacilityManagementWeb/SmartHotel.FacilityManagementWeb/ClientApp/src/app/app.component.html

```diff
@@ -1,12 +1,13 @@
 <!--The content below is only a placeholder and can be replaced.-->
-<div class='container-fluid'>
+<div>
 
-  <div class='row content'>
-    <router-outlet></router-outlet>
-  </div>
-  <div class='row fixed-header'>
+  <div class='fixed-header'>
     <app-nav-menu></app-nav-menu>
   </div>
+  <div>
```

Comment:
@PiotrKarczmarz The positioning of having the app-nav-menu defined AFTER the router-outlet is needed. (Maybe not in the future with the way we're changing up the Rooms screen, but we should keep this protection in place)

Once the scrolling issue of the header not staying in the same place is fixed, if you navigate to the the "rooms" screen, make sure that the browser is smaller and has vertical scrolling, and then scroll down. Without this setup, parts of the slider controls will possibly show up on top of the header when scrolling

Reply:
It works well now (commit: 7db7fdf1b3c148cb160b22ddf959c8bcc29d1064) when app-nav-menu is first. The rooms page with controls are below the header when scrolling.